src/CmsInterfaceManagerBundle/objects/db/TCMSInterfaceManager/TCMSInterfaceManagerBaseExportCSV.class.php

```diff
@@ -100,6 +104,9 @@ protected function PrepareImport()
      * this method is always called at the end of RunImport (even if the import failed) to do any cleanup work.
      *
      * @param bool $bImportSucceeded - set to true if the import succeeded
+     *
+     * @FIXME Parent return `$bImportSucceeded` (bool). This method should probably do the same?
```

Comment:
This child method changes the return type from `bool` to `void`. This method should probably also return `$bImportCusseeded` at the end in order to be consistent and compatible with it's parent.